pkg/gadgets/internal/test/runner.go

```diff
@@ -0,0 +1,180 @@
+//go:build linux
+// +build linux
+
+// Copyright 2022 The Inspektor Gadget authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package test
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+	"runtime"
+	"syscall"
+
+	"golang.org/x/sys/unix"
+)
+
+// Config defines how the runner should behave.
+// TODO: We could implement more options like unsharing the network
+// namespace, running on a different group ID, etc.
+type Config struct {
+	// User ID to run under
+	UID int
+}
+
+type Info struct {
```

Comment:
I know those structs are in `runner.go` file, but from the documentation perspective, source location in file does not matter so those struct will show up as `test.Config` and `test.Info`. I think it would be cleaner if they would be named as `RunnerConfig` and `RunnerInfo`, as as far as I understand, this is what they actually represent.